components/brave_new_tab_ui/data/LICENSE

```diff
@@ -1,63 +1,59 @@
-ntp-1.webp:
-URL: https://www.pexels.com/photo/red-white-and-blue-abstract-painting-1045850/
-License: https://www.pexels.com/photo-license/
-
-ntp-2.webp:
-URL: https://unsplash.com/photos/zjyP-UYI-ko
+anders-jilden.webp:
+URL: https://unsplash.com/photos/uwbajDCODj4
 License: https://unsplash.com/license
 
-ntp-3.webp:
-URL: https://unsplash.com/photos/E4944K_4SvI
+andreas-gucklhorn.webp:
+URL: https://unsplash.com/photos/mawU2PoJWfU
 License: https://unsplash.com/license
 
-ntp-4.webp:
-URL: https://unsplash.com/photos/eXHeq48Z-Q4
+andy-mai.webp:
+URL: https://unsplash.com/photos/4knR_YzeUVc
 License: https://unsplash.com/license
 
-ntp-5.webp:
-URL: https://www.flickr.com/photos/spacex/16510243060/
-License: https://creativecommons.org/licenses/by-nc/2.0/
+annie-spratt.webp:
+URL: https://unsplash.com/photos/C9jP5AlgcuQ
+License: https://unsplash.com/license
 
-ntp-6.webp:
-URL: https://www.flickr.com/photos/spacex/16236321533/
-License: https://creativecommons.org/licenses/by-nc/2.0/
+anton-repponen.webp:
+URL: https://unsplash.com/photos/sfgH9dXcMRw
+License: https://unsplash.com/license
 
-ntp-7.webp:
-URL: https://www.flickr.com/photos/spacex/40126461851/
-License: https://creativecommons.org/licenses/by-nc/2.0/
+ben-karpinski.webp:
+URL: Contributor sent the hi-res version through email
+License: Used with permission
 
-ntp-8.webp:
-URL: https://www.instagram.com/p/BZRQxf9hRpw/
-License: used with permission
+dc-cavalleri.webp:
+URL: Contributor sent the hi-res version through email
+License: Used with permission
 
-ntp-9.webp:
-URL: https://www.instagram.com/p/BI7w_dFAlR0/
-License: used with permission
+joe-gardner.webp:
+URL: Contributor sent the hi-res version through email
```

Comment:
According to https://github.com/brave/brave-browser/issues/6904#issuecomment-559242794, that should be https://unsplash.com/photos/4xv3lqnanYc and the Unlicense license.